framework/src/iotbus/Kconfig

```diff
@@ -18,6 +18,15 @@ config IOTBUS_GPIO
 	---help---
 		Enables IoTbus GPIO
 
+if IOTBUS_GPIO
+	config IOTBUS_GPIO_INTERRUPT_PRIORITY
```

Comment:
It is priority of thread, isn't it?
Currently it seems that priority of interrupt itself....
Also 224 is same as CONFIG_SCHED_HPWORKPRIORITY, you'd better add comment regarding it.